### DISCUSSION TITLE
Linkage into PIC foundry specific process development kits (PDK) ?

### DISCUSSION THREAD

**Discussion by mniehus** [HelgeGehring/gdshelpers#55] [General]
Hi (again :) what would be the way forward in using gdshelper towards PIC foundry specific process development kits (PDK) ? any ideas/thoughts/suggestions ? thank You. cheers.

**Comment by HelgeGehring** [comment]
Hey :) How is your PDK defined? Just by GDSII-files? Those you can import using the part "GDSIIImport", which returns shapes which you can place in the layout. To make it more comfortable it might make sense to add a new PDK-Part, which allows to add ports/... to make the handling more comfortable. Would be awesome if you could share some details about what is needed! :)
